front/src/components/header/Header.tsx

```diff
@@ -93,7 +91,7 @@ const Header = ({ isLoggedIn, setIsLoggedOut }: Props) => {
         mr={1}
         mt={{ base: 4, md: 0 }}
         padding={{ base: 0, md: '0 12px 5px' }}
-        width={{ base: mobileWidth, md: 'unset' }}
+        width={{ base: 0, md: 'unset' }}
```

Comment:
I think we can remove this whole line, so we don't even have width here at all, if you don't mind.

Reply:
Yes, that's even better

Reply:
Also some changes went in that change 'padding' to 'p' so if you see a merge conflict that's why.

Reply:
Not trying to rush :) but you will remove the widths lines right?